scripts/override-start.js

```diff
@@ -24,8 +24,28 @@ const overrideWebpackConfig = ( config, devServer ) => {
 	// Replace the react-dev-utils webpackHotDevClient with a version patched to
 	// correctly detect the dev server host & port for socket requests.
 	const hotClient = require.resolve( 'react-dev-utils/webpackHotDevClient' );
-	const hotClientIndex = config.entry.indexOf( hotClient );
-	config.entry.splice(hotClientIndex, 1, require.resolve( '../overrides/webpackHotDevClient' ) );
+
+	if ( config.entry instanceof Array ) {
+		const hotClientIndex = config.entry.indexOf( hotClient );
+		config.entry.splice( hotClientIndex, 1, require.resolve( '../overrides/webpackHotDevClient' ) );
+	}
+
+	if ( config.entry instanceof Object ) {
+		Object.keys( config.entry ).forEach( key => {
```

Comment:
This looks like it should be `Object.entries`.

Reply:
Sometimes forget about node environment vs web! Thought `Object.entries` was still experimental so I've not made a habit of using it